### ISSUE TITLE
Some armor blocks (slabs, wedges) don't have connected textures

### ISSUE THREAD

**Issue by ViciousVinnyD** [KeenSoftwareHouse/SpaceEngineers#601]
Some armor blocks including slabs and the underside of slopes have problems with the texture which would normally continue between blocks but instead sharply cuts off between blocks.

Another thing that bothers me is how some armor blocks (slabs for example) don't have that metal trim at the edges of the block, making them look out of place when used with regular blocks. 
Round blocks do have that metal trim, but it doesn't go away when you use them to make cylinders.

<a href="https://imgur.com/H4u6pRo"><img src="https://i.imgur.com/H4u6pRo.jpg" title="source: imgur.com" /></a>
The armor cubes to the right have a nice, contiuing texture. The slabs and ramps to the left cut off sharply between blocks.

TL;DR - Please make armor blocks more coherent and asthetically pleasing.

**Comment by mccorkle** [comment]
@ViciousVinnyD This is not the right place to report issues or bugs to keen.  Please go to their support forum at https://support.keenswh.com/spaceengineers/general
